docs/projects/ad4134_fmc/index.rst

```diff
@@ -0,0 +1,252 @@
+=============== === ========== ===========
+axi_ad4134_dma  13  57         89
+spi_ad4134      12  56         88
+=============== === ========== ===========
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here </>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/ad4134_fmc/zed
+   user@analog:~/hdl/projects/ad4134_fmc/zed$ make
+
+A more comprehensive build guide can be found in the :ref:`build_hdl` user guide.
+
+Resources
+-------------------------------------------------------------------------------
+
+Hardware related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Product datasheet: :adi:`AD4134`
+
+-  `UG-2016, EVAL-AD4134FMCZ Board User Guide <https://www.analog.com/media/en/technical-documentation/user-guides/eval-ad4134-ug-2016.pdf>`__
+
+HDL related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-hdl:`AD4134-FMC HDL project source code <projects/ad4134_fmc>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_clkgen`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac`
+     - :ref:`here <axi_dmac>`
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine`
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution`
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect`
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload`
+     - :ref:`here <spi_engine offload>`
+
+-  :ref:`SPI Engine Framework documentation <spi_engine>`
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Linux support**
+
+- linux driver: `ad4134.c <https://github.com/analogdevicesinc/linux/blob/master/drivers/iio/adc/ad4134.c>`__
+- device tree: `zynq-zed-ad4134.dts <https://github.com/analogdevicesinc/linux/blob/master/arch/arm/boot/dts/zynq-zed-adv7511-ad4134.dts>`__
+
+More information about linux drivers: :dokuwiki:`AXI ADC HDL Linux  [Wiki] <resources/tools-software/linux-drivers/iio-adc/axi-adc-hdl>`
```

Comment:
Instead of the generic AXI ADC HDL Linux I would add the actual driver's wiki page, if exists. Example:
https://github.com/analogdevicesinc/hdl/blob/main/docs/projects/ad9081_fmca_ebz/index.rst?plain=1#L760